src/scripts/main.js

```diff
@@ -1,5 +1,28 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  if (e.target.className === 'wall') {
+    const x = e.clientX;
+    const y = e.clientY;
+    const field = e.target.getBoundingClientRect();
+    const spider = document.querySelector('.spider');
```

Comment:
You're querying the DOM for the .spider element every time a click event is fired. Querying the DOM is a costly operation and should be avoided whenever possible. You could improve the performance of your code by querying for the .spider element once and storing it in a variable outside of the event listener.